utopia-remix/app/handlers/validators.ts

```diff
@@ -32,7 +34,20 @@ export function validateProjectAccess(
     const isCreator = userId ? ownerId === userId : false
 
     const allowed = isCreator || (await hasUserProjectPermission(projectId, userId, permission))
-    ensure(allowed, errorMessage ?? 'Unauthorized Access', status ?? Status.UNAUTHORIZED)
+    let errorMessageToUse = errorMessage ?? 'Unauthorized Access'
+    let statusToUse = status ?? Status.UNAUTHORIZED
+    if (!allowed && canRequestAccess) {
```

Comment:
Wait what? Why the `=== true`?

Reply:
I don't want to go dig for past comments (veeeeeery old), but the concept I was told is to avoid doing bool comparisons for stuff that can be null/undefined :)

Reply:
Ah that would have been about avoiding truthiness. In this case though it's unnecessary since we're providing a default value, so `canRequestAccess` can't be `undefined`